utils/zerovec/src/ule/option.rs

```diff
@@ -54,6 +54,15 @@ impl<U: Copy> OptionULE<U> {
     }
 }
 
+impl<U> core::fmt::Debug for OptionULE<U> {
+    fn fmt(&self, f: &mut core::fmt::Formatter<'_>) -> core::fmt::Result {
+        f.debug_tuple("OptionULE")
```

Comment:
ditto, this should format the non-ULE type

Reply:
I can't find a way to make it work. Can you advise?